schema/me/__tests__/follow_gene.test.js

```diff
@@ -0,0 +1,51 @@
+import { runAuthenticatedQuery } from "test/utils"
+
+describe("FollowGene", () => {
+  it("follows a gene", () => {
+    const mutation = `
+      mutation {
+        followGene(input: { gene_id: "pop-art" }) {
+          gene {
+            id
+            name
+          }
+        }
+      }
```

Comment:
Its funny how straight forward mutations are compared to what I used to have in my mind. Did they become simpler at some point? I remember there used to be an `input` type and a lot of strange ceremony, but yah! so nice.

Reply:
The "input" and strange ceremony (the mutation ID etc) was a requirement of "classic relay"